tests/App.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { mount, render, shallow } from 'enzyme';
```

Comment:
Some of your imports are not being used. I think it's better to remove them

Reply:
okay, i can remove it